prompb/types.proto

```diff
@@ -23,10 +23,17 @@ message Sample {
   int64 timestamp = 2;
 }
 
+message Exemplar {
+  repeated Label labels = 1 [(gogoproto.nullable) = false];
+  double value = 2;
+  int64 timestamp = 3;
```

Comment:
How is this being missing represented?

How are you tying this to the sample it was reported with?

Reply:
>How is this being missing represented?

in the scrape/storage code if there wasn't a timestamp parsed from the exemplar text I give it the scrape timestamp (same as the sample would get)

> How are you tying this to the sample it was reported with?

Not entirely sure what you're asking here.

Reply:
> in the scrape/storage code if there wasn't a timestamp parsed from the exemplar text I give it the scrape timestamp

So you're not representing whether there was an exemplar timestamp exposed. I think you need to think more about the semantics here.

Reply:
See my response to the other timestamp related thread; I'll separate the stored scrape timestamp from the exposed exemplar timestamp.